crypto_automations/automation/__init__.py

```diff
@@ -0,0 +1,18 @@
+#  This file is part of Crypto-Automations (https://github.com/Drakkar-Software/Crypto-Automations)
+#  Copyright (c) 2023 Drakkar-Software, All rights reserved.
```

Comment:
```suggestion
#  Copyright (c) 2024 Drakkar-Software, All rights reserved.
```

Reply:
(I know we need to update the other repo as well but we can as well start with 2024 here)

Reply:
:open_mouth: that's true